src/tests/ftest/io/llnl_mpi4py.yaml

```diff
@@ -17,6 +17,9 @@ pool:
     control_method: dmg
 client_processes:
     np: 8
+container:
+#                         [type: enable_chksum: srv_verify: chksum_type: chunk_size]
+    container_properties: [POSIX, False, False, 0, 1048576]
```

Comment:
i don't think this is really creating a container with type POSIX. the failures indicate an invalid error from dfs_mount, which usually means it's an invalid container type.
Is this tested somewhere else? only place i see is ior_small.yaml, which is using the DAOS driver which doesn't actually require / check for POSIX container. only the DFS driver does.

Reply:
for now i made a PR to just use the tool:
https://github.com/daos-stack/daos/pull/2164